opty/direct_collocation.py

```diff
@@ -166,6 +172,13 @@ def __init__(self, equations_of_motion, state_symbols,
             ndarrays of floats of shape(N,). Any time varying parameters in
             the equations of motion not provided in this dictionary will
             become free trajectories optimization variables.
+        known_experiment_map : dictionary, optional
```

Comment:
This known_experiment_map is the experimental data we need use in the extra term and I want to write a explanation for that. I am not sure how to write this explanation, so I wrote this according to known_trajectory_map. (might be wrong)